pkg/build/webhook/webhook.go

```diff
@@ -3,9 +3,12 @@ package webhook
 import (
 	"crypto/hmac"
 	"errors"
+	"net/http"
 	"strings"
 
-	"github.com/openshift/origin/pkg/build/api"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
```

Comment:
got a couple of places where i use api.build, should i rename it to buildapi? is that a convention, or is it used here just because kapi is as well and you want a to differentiate between them more clearly?